client/src/components/Button/index.js

```diff
@@ -0,0 +1,23 @@
+import React, { Component } from 'react';
+import { Link } from "react-router-dom";
+import btnimg from '../../btn.jpg';
+
+
+class Button extends Component {
+  state = {  }
```

Comment:
If you don't need a state, don't declare it. Also, this could be a functional component since it doens't handle any logic.